src/Features/CSharp/Portable/AddBraces/CSharpAddBracesCodeFixProvider.cs

```diff
@@ -31,7 +31,7 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
             return Task.CompletedTask;
         }
 
-        protected override Task FixAllAsync(
+        internal override Task FixAllAsync(
             Document document, ImmutableArray<Diagnostic> diagnostics, 
             SyntaxEditor editor, CancellationToken cancellationToken)
```

Comment:
This was necessary due to us switching to jsut using the normal SyntaxEditorBasedCodeFixProvider for the embedded language code-fix-provider system.  Doing that means we wanted to aggreate many CodeFixProviders into one final exported one. So that one exported one needed to be able to call into this on all the individual instances.  so it now needs to be internal.